youtube_dl/extractor/openload.py

```diff
@@ -64,16 +64,17 @@ def _real_extract(self, url):
             raise ExtractorError('File not found', expected=True)
 
         ol_id = self._search_regex(
-            '<span[^>]+id="[a-zA-Z0-9]+x"[^>]*>([0-9]+)</span>',
+            '<span[^>]+?id="[^"]*?"[^>]*?>([^<]+?)<\/span>',
```

Comment:
i usually work in php so i'm used to of escaping slashes that ain't required ! nor the question marks